docs/src/modules/java-protobuf/pages/views.adoc

```diff
@@ -321,8 +321,28 @@ include::example$scala-protobuf-eventsourced-customer-registry/src/main/scala/cu
 [#topic-view]
 == Creating a View from a topic
 
-The source of a View can be an eventing topic. You define it in the same way as shown in <<event-sourced-entity>> or <<value-entity>>, but leave out the `eventing.in` annotation in the Protobuf file.
+The source of a View can be an eventing topic. You define it in the same way as shown in <<event-sourced-entity>> or <<value-entity>> but the View definition must set `eventing.in.topic` to a given topic name. Note that, on your producer side you must manually pass the `ce-subject` metadata, required by the View component.
```

Comment:
I would probably be clear here that the example below is about producing to topic with metadata and not about "creating a view from a topic".

```suggestion
The source of a View can be an eventing topic. You define it in the same way as shown in <<event-sourced-entity>> or <<value-entity>> but the View definition must set `eventing.in.topic` to a given topic name. Note that, on your producer side you must manually pass the `ce-subject` metadata, required by the View component. See example below for how to pass such metadata.
```

Reply:
good idea, added